micah-akpan/turing_shopping_cart
# [Chore #168721469] Configure and setup Dev environment

#### What does this PR do?
This PR configures and setups databases and other minor requirements needed to aid development
#### Description of Task to be completed?
- Configure and set-up DBs for testing and dev environment
#### How should this be manually tested?
- Git fetch and checkout to chore/setup-dev-test-env
- Provided you have a working installation of MySQL server, run this with the MySQL client: `mysql -u <your_dbuser> -D <your_databasename> -p < ./src/database/database.sql`
- Data should be migrated and saved in the DB
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168721469](https://www.pivotaltracker.com/story/show/168721469)
#### Screenshots (if appropriate)
N/A
#### Questions:
